docs/package_templates/autotools_package/all/conanfile.py

```diff
@@ -78,11 +77,12 @@ def configure(self):
         self.settings.rm_safe("compiler.libcxx")
 
     def layout(self):
-        # src_folder must use the same source folder name the project
```

Comment:
please, keep this comment, it explains why do we use src_folder="src"

Reply:
A comment would not hurt, but the current wording is kind of broken, imo. Could you provide a better one, maybe?